src/main/java/seedu/address/model/group/GroupName.java

```diff
@@ -20,7 +23,8 @@ public class GroupName {
      * The first character of the address must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "^(?i)CS2103T?-[A-Z]\\d+-\\d+$";
+    public static final String VALIDATION_REGEX = "^(?i)(?<coursepart>CS2103T?)-(?<tutorialpart>[A-Z]\\d+)-"
```

Comment:
Just to confirm, the validation regex is still the same, but when formatting the group name it removes any leading 0s?

Reply:
yep, the regex is the same.

i just added the capture groups for easy stripping